### DISCUSSION TITLE
system-wide mobileconfig for Mac

### DISCUSSION THREAD

**Discussion by csousougen** [kkebo/DNSecure#27] [Ideas]
Hey, thanks for your nice work!

Since your app is not available for Mac yet, could you please share a mobileconfig that could be used in Mac?
I need a profile that is system-wide and not used only by Safari. I cannot find it elsewhere.

Thanks

**Comment by kkebo** [comment]
Thank you for an idea.

To be clear, `NEDNSSettingsManager` that DNSecure uses is system-wide and macOS supports it. So, It doesn't necessarily have to use mobileconfig.

I've been trying to support macOS with `NEDNSSettingsManager`, but haven't achieved yet, so that might be a good idea.

**Reply by kkebo** [reply]
I'm working on it in #34.
